### PR TITLE
[stable/openvpn] #1785 namespace defined templates with chart name

### PR DESCRIPTION
**Pull request by kevinschumacher**
#1785

### CODE REVIEW

File: stable/openvpn/Chart.yaml [hunk 1]
```diff
@@ -1,22 +1,23 @@
 apiVersion: v1
-description: A Helm chart to install an openvpn server inside a kubernetes cluster.  Certificate generation is also part of the deployment, and this chart will generate client keys as needed.
+description: A Helm chart to install an openvpn server inside a kubernetes cluster.  Certificate
+  generation is also part of the deployment, and this chart will generate client keys
+  as needed.
 name: openvpn
-version: 1.1.2
+version: 2.0.0
```
Comment (dhilipkumars) [inline stable/openvpn/Chart.yaml:6]: Please update only the minor version

**Review by dhilipkumars — CHANGES_REQUESTED** [review]

File: stable/openvpn/Chart.yaml (hunk 1 above)
Comment (kevinschumacher) [reply stable/openvpn/Chart.yaml:6]: IMO this is a breaking change to the chart's API. 

[A previous discussion on a similar PR for stable/acs-engine-autoscaler](https://github.com/kubernetes/charts/pull/1976#discussion_r136875811)

If this is a breaking API change, then I think we should be bumping the major version.

**Comment by jfelten** [comment]
@kevinschumacher can you explain why you think this is a change to the chart's api?  I read https://github.com/kubernetes/charts/issues/1785, but don't quite see it.   Are you referring to other charts that use openvpn as a subchart and reference the templates?

**Comment by mattfarina** [comment]
@jfelten I think the sub-chart case is what is meant. If someone uses openvpn as a sub-chart and their values files need to change in any way that breaks backwards compatibility it's an API breaking change. In SemVer that's a major version bump.

**Comment by jfelten** [comment]
Thanks @mattfarina - I agree that anything breaking downstream dependencies = api change, so the version bump is justified.

**Review by dhilipkumars — APPROVED** [review]

**Pull request merged**